MIP104/MIP104.md

```diff
@@ -265,8 +265,8 @@ Spark:
 - `K` = 15.53%
 - `Ka` = 0.00%
 - `Kb` = 30.00%
-- `KFa` = 0.0015%
-- `KFb` = 0.07%
+- `KFa` = 6%
+- `KFb` = 34%
```

Comment:
Let's add 2 decimal places to make it consistent with other entries such as line 267

```suggestion
- `KFa` = 6.00%
- `KFb` = 34.00%
```